### PR TITLE
EDMG BNF

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Minor changes in mss, stub for Griaule 32.

### CODE REVIEW

File: EndaMaryamDengelat/EDMG045.xml [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+                    </msContents>
+                    <physDesc>
+                       
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="127">27</measure>
```
Comment (CarstenHoffmannMarburg) [inline EndaMaryamDengelat/EDMG045.xml:42]: Number in quantity should match the number in the tag.

Is it by accident the same value as in DHAS024 above, or possibly a Copy&Paste error?

File: EndaMaryamDengelat/EDMG045.xml [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+                        </msIdentifier>
+                        <msContents>
+                            <summary/>            
+                            <msItem xml:id="p4_i1">
+                                <locus from="25ra" to="26vb"/>
+                                <title type="incomplete" ref="LIT1960Mashaf">Missal (fragment)</title>
+                                <textLang mainLang="gez"/>   
+                  
+                                <notatedMusic>
+                                    <desc>Musical notation in red and black thoughout the text.</desc>
```
Comment (CarstenHoffmannMarburg) [inline EndaMaryamDengelat/EDMG045.xml:276]: throughout

File: ParisBNF/abb/BNFabb57.xml [hunk 3]
```diff
@@ -103,10 +101,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
               <title type="complete" ref="LIT5798ListKingsIsrael">ነገሥት᎓ እሉ᎓ ነግሡ᎓ ለእስራኤል᎓ እሉ᎓
                 እሙንቱ᎓</title>
               <incipit xml:lang="gez"/>
-              <explicit xml:lang="gez">ቀዳሚ᎓ ኵሉ᎓ ነግሠ᎓ ሳኦል᎓ ለእስራኤል᎓ ዓመታት᎓ ፵ዘእም᎓ ነገደ᎓ ብንያም᎓ ወእምነገደ᎓
-                ይሁዳ፨ ወእምድኅረዝ᎓ ነግሠ᎓ ዳዊት᎓</explicit>
-              <incipit xml:lang="gez">ወዛካርያስ᎓ ወፈጸምያነ᎓ ትንቢት᎓ ከዊኖሙ᎓ ወዳንኤልሰ᎓ ወሕዝቅኤልሰ᎓ በፄዋዌ᎓ ተነበዩ᎓ በቢሎን᎓
-                ወኮኑ᎓ ካህናተ᎓ ኵሎሙ᎓ <hi rend="rubric">፯፡</hi>
+              <explicit xml:lang="gez">ቀዳሚ፡ ኵሉ፡ ነግሠ፡ ሳኦል፡ ለእስራኤል፡ ዓመታት፡  ፵ዘእም፡ ነገደ፡ ብንያም፡ ወእምነገደ፡ ይሁዳ፨ ወእምድኅረዝ፡ ነግሠ፡ ዳዊት፡</explicit>
+              <incipit xml:lang="gez">ወዛካርያስ፡ ወፈጸምያነ፡ ትንቢት፡ ከዊኖሙ፡ ወዳንኤልሰ፡ ወሕዝቅኤልሰ፡ በፄዋዌ፡ ተነበዩ፡ በቢሎን፡ ወኮኑ፡ ካህናተ፡ ኵሎሙ፡ <hi rend="rubric">፯፡</hi>
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb57.xml:105]: Why did you place the incipit after the explicit? Will this be presented on the website in the same order or is the order always, that the incipit is placed before the explicit after it is transformed, as it should be?

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: EndaMaryamDengelat/EDMG045.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply EndaMaryamDengelat/EDMG045.xml:42]: Just a mistake.

File: EndaMaryamDengelat/EDMG045.xml [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+                     This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </licence>
+               </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1017EndaMaryamDengelat"/>
+                        <idno>EDMG-045</idno>      
+                        <idno facs="EDMG/045/EDMG-045_" n="58">EDMG-045</idno>
```
Comment (eu-genia) [inline EndaMaryamDengelat/EDMG045.xml:29]: I do not have these images I believe

File: EndaMaryamDengelat/EDMG045.xml [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+                  <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                     This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </licence>
+               </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1017EndaMaryamDengelat"/>
+                        <idno>EDMG-045</idno>
```
Comment (eu-genia) [inline EndaMaryamDengelat/EDMG045.xml:28]: why two idno?

File: EndaMaryamDengelat/EDMG045.xml [hunk 6]
```diff
@@ -0,0 +1,429 @@
...
+                  </licence>
+               </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1017EndaMaryamDengelat"/>
+                        <idno>EDMG-045</idno>      
+                        <idno facs="EDMG/045/EDMG-045_" n="58">EDMG-045</idno>       
+                        </msIdentifier>
```
Comment (eu-genia) [inline EndaMaryamDengelat/EDMG045.xml:30]: please also check EDMG001 EDMG002 etc that have multiple `idno `elements - there should only be one, all others should be in `altIdentifier`

File: EndaMaryamDengelat/EDMG045.xml [hunk 7]
```diff
@@ -0,0 +1,429 @@
...
+                        <msContents>
+                            <summary/>            
+                            <msItem xml:id="p5_i1">
+                                <locus from="27ra" to="27vb"/>
+                         
+                                <title type="incomplete">Collection of salām-hymns</title>
+                                <textLang mainLang="gez"/>   
+                                <note>Only two thirds of the text leaf, originally belonging probably to 
+                                    Ms <ref type="mss" corresp="DHAS023">Dabra Hāllelo ʾƎndā ʾAbuna Sāmuʾel DHAS-023</ref>.</note>
+                            </msItem>
```
Comment (eu-genia) [inline EndaMaryamDengelat/EDMG045.xml:342]: I think we need a relation to show this

File: ParisBNF/et/BNFet336.xml [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>                       
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>Griaule</collection>
+                        <idno>BnF Éthiopien 336 (Griaule 32)</idno>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet336.xml:30]:
```
                        <idno>BnF Éthiopien 336</idno>
                        <altIdentifier>
                            <idno>Griaule 32</idno>
                        </altIdentifier>
```
you do not need Griaule 32 twice

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: EndaMaryamDengelat/EDMG045.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply EndaMaryamDengelat/EDMG045.xml]: ok

**Review by nafisa-valieva — APPROVED** [review]
Thank you! Looks good for me!

**Review by nafisa-valieva — APPROVED** [review]

File: EndaMaryamDengelat/EDMG045.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply EndaMaryamDengelat/EDMG045.xml:30]: Changed.

File: EndaMaryamDengelat/EDMG045.xml (hunk 7 above)
Comment (DenisNosnitsin1970) [reply EndaMaryamDengelat/EDMG045.xml:342]: changed.

File: EndaMaryamDengelat/EDMG045.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply EndaMaryamDengelat/EDMG045.xml:28]: changed.

File: EndaMaryamDengelat/EDMG045.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply EndaMaryamDengelat/EDMG045.xml:29]: They are coming, need some processing.

File: ParisBNF/et/BNFet336.xml (hunk 8 above)
Comment (eu-genia) [reply ParisBNF/et/BNFet336.xml:30]: We should always be very careful to keep **encoding** and **visualization** apart.
Each identifier and each shelfmark MUST be encoded separately.
If for some reason the visualization is not what you want then you need an issue for visualization. We can always have a different way to show altIdentifiers, or we can have separate rules for different types, this is all possible, but encoding has nothing to do with visualization.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**